linter_exclusions.yml

```diff
@@ -231,6 +231,16 @@ eventgrid event-subscription update:
     endpoint_type:
       rule_exclusions:
       - no_parameter_defaults_for_update_commands
+eventgrid partner topic event-subscription update:
```

Comment:
Discussed with service team offline. It would bring a breaking change. Meanwhile, the logic behind three update are same. It need some time to fix these violations.